lib/msf/core/handler/reverse.rb

```diff
@@ -82,12 +83,28 @@ def setup_handler
                 'MsfPayload' => self,
                 'MsfExploit' => assoc_exploit
               })
+
+          #Checking whether LHOST is a loopback address
+
+          def is_loopback_address(address)
```

Comment:
Move this method to the module level so you can use it in `Msf::Handler::ReverseHttp`.